File: API/BugTracker/Models/Project.cs
namespace BugTracker.Models
{
    public class Project
    {
        public int Id { get; set; } 
        public string Name { get; set; }
        public string Description { get; set; }

        //project-user: many-to-many
        public ICollection<ProjectUser> ProjectUsers { get; set; }
        public ICollection<Ticket> Tickets { get; set; } 
    }
}

File: Client/src/app/_models/project.ts
// import { Ticket } from "./ticket"

import { Member } from "./member"

export interface Project{
  id: number
  name: string
  description: string
  tickets: Ticket[]
}

export interface Ticket {
  id: number
  title: string
  created: Date
  assignedDeveloper: Member
  submitter: Member
  projectId: number
  description: string
  status: string
  priority: string
}
